.env.example

```diff
@@ -32,6 +32,9 @@ S3_PREFIX=
 S3_ACCESS_KEY=minioadmin
 S3_SECRET_KEY=minioadmin
 
+S3_START_PATH=annotation
```

Comment:
What exactly these `S3_START_PATH` and `S3_TEXT_PATH` are doing? Is it required for all services?

Reply:
these are folder names in Minio. I guess if service works directly with S3 it should have known it.